soc/arm/microchip_mec/mec172x/Kconfig.defconfig.series

```diff
@@ -19,4 +19,10 @@ source "soc/arm/microchip_mec/mec172x/Kconfig.defconfig.mec172x*"
 config CORTEX_M_SYSTICK
 	default y
 
+config CLOCK_CONTROL_MCHP_XEC
+	default y
+	depends on CLOCK_CONTROL
+	help
+	  Enable support for Microchip XEC clock controller driver.
```

Comment:
clock-controller driver

Can you add a bit of help about what features the driver has? Kconfig help is supposed to be 4+ lines as I understand it

Reply:
Done